integration-tests/hibernate-reactive-orm-compatibility/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=hibernate_orm_test
+quarkus.datasource.password=hibernate_orm_test
+
+# Hibernate config
+#quarkus.hibernate-orm.log.sql=true
+quarkus.hibernate-orm.database.generation=drop-and-create
+
+# Reactive config
+quarkus.datasource.reactive=true
+quarkus.datasource.reactive.url=${postgres.reactive.url}
+
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5431/hibernate_orm_test
+
+
+quarkus.security.users.embedded.enabled=true
+quarkus.security.users.embedded.users.scott=jb0ss
+quarkus.security.users.embedded.plain-text=true
+quarkus.security.users.embedded.roles.scott=Admin,admin,Tester,user
```

Comment:
Is this relevant?

Reply:
It's in most integration tests and apparently it's needed, if you remove it it fails. TBH I haven't investigated why

Reply:
Looks like this is only necessary because of the `@Authenticated` and elytron dependencies you copied over from another module. But IMO it's highly questionable whether these should have been in the other module in the first place (see https://github.com/quarkusio/quarkus/commit/e54abc31683a04d214a8652dc2b3fc5dc2057862#diff-03a6bffae5379c212bc588672d3271b83585afaa8c98fc21461298c8588017dd), and in any case I don't think it makes sense to use `@Authenticated` and elytron in this new module.

Reply:
Removed everything thank you for checking out :)